site/content/en/docs/contributing/development-environment.md

```diff
@@ -152,25 +152,20 @@ description: 'Installing a development environment for different operating syste
   >
   > Perform this action before installing cvat requirements from the list mentioned above.
 
-- Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) and [Docker-Compose](https://docs.docker.com/compose/install/)
+- Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) and [Docker Compose](https://docs.docker.com/compose/install/)
 
-- Pull and run Open Policy Agent docker image:
+- Start service dependencies:
 
   ```bash
-   docker run -d --rm --name cvat_opa_debug -p 8181:8181 --network=host openpolicyagent/opa:0.45.0-rootless \
-   run --server --set=decision_logs.console=true --set=services.cvat.url=http://localhost:7000 \
-   --set=bundles.cvat.service=cvat --set=bundles.cvat.resource=/api/auth/rules
+  docker compose -f docker-compose.yml -f docker-compose.dev.yml up -d --build \
+    cvat_opa cvat_db cvat_redis cvat_server
   ```
 
-- Pull and run PostgreSQL docker image:
+  Note: this runs an extra copy of the CVAT server in order to supply rules to OPA.
```

Comment:
It should also be mentioned that when OPA rules are changed, the CVAT server container must be rebuilt and restarted to apply the new rules.

Reply:
Alright, updated.